### PR TITLE
Wire invoice resolution

### PR DESCRIPTION
**Pull request by benrudolph**
@esoergel this should migrate all the InvoicePdfs as well make sure any new ones that are created properly have the `is_wire` field. There are a total of 14 wire invoices on prod, so IterDB is a bit of overkill, but was neat to use!
http://manage.dimagi.com/default.asp?177208#984768
buddy: @orangejenny

### CODE REVIEW

File: corehq/apps/accounting/management/commands/migrate_wire_invoice_pdfs.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from optparse import make_option
```
Comment (dimagimon) [inline corehq/apps/accounting/management/commands/migrate_wire_invoice_pdfs.py]: F401 'make_option' imported but unused

File: corehq/apps/accounting/management/commands/migrate_wire_invoice_pdfs.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    help = ("Migrate InvoicePdfs to have is_wire field")
+
+    def handle(self, *args, **options):
+        invoice_ids = WireBillingRecord.objects.values_list('pdf_data_id', flat=True)
+        db = InvoicePdf.get_db()
+
+        with IterDB(db) as iter_db:
+            for doc in iter_docs(db, invoice_ids):
+                doc['is_wire'] = True
+                iter_db.save(doc)
```
Comment (esoergel) [inline corehq/apps/accounting/management/commands/migrate_wire_invoice_pdfs.py]: Not actionable, but FYI you can also do this as
```python
from corehq.util.couch import iter_update, DocUpdate

def update_invoice(doc):
    doc['is_wire'] = True
    return DocUpdate(doc)

result = iter_update(db, update_invoice, invoice_ids, verbose=True)
```
And that'll do things like retry if there's a error, skip documents that are unchanged, and give some info in the output (if `verbose == True`).

**Comment by esoergel** [comment]
:+1:

File: corehq/apps/accounting/management/commands/migrate_wire_invoice_pdfs.py (hunk 2 above)
Comment (benrudolph) [inline corehq/apps/accounting/management/commands/migrate_wire_invoice_pdfs.py]: cool, good to know

**Pull request merged**